examples/content_management_examples/common/examples/todos/todos.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+import {
+  CreateIn,
+  DeleteIn,
+  GetIn,
+  SearchIn,
+  UpdateIn,
+} from '@kbn/content-management-plugin/common';
+
+export const TODO_CONTENT_ID = 'todos';
+export interface Todo {
+  id: string;
+  title: string;
+  completed: boolean;
+}
+const todoSchema = schema.object({
+  id: schema.string(),
+  title: schema.string(),
+  completed: schema.boolean(),
+});
+
+export type TodoCreateIn = CreateIn<'todos', { title: string }>;
+export type TodoCreateOut = Todo; // TODO: Is this correct?
```

Comment:
Because we don't control what comes out from the CRUD + search services. It could be a string, it could be an object, it could be an array. Every content will have their own out interface from a `storage.get()` operation

Reply:
Looking at it again we could expose a helper interface for the create out. I would be a combination of the RPC response 

```ts
interface RpcResponse<T> {
  result: T
}
```

with the Storage layer response (e.g. https://github.com/elastic/kibana/blob/main/src/plugins/content_management/server/core/crud.ts#L21)

```ts
export interface CreateItemResponse<T = any> {
  contentTypeId: string;
  result: T; // own by each team
}
```

So probaby something like this

```
export type CreateOut<T> = RpcReponse<CreateItemResponse<T>>;
```